sharding-proxy/sharding-proxy-backend/src/test/java/org/apache/shardingsphere/shardingproxy/backend/schema/LogicSchemaTest.java

```diff
@@ -0,0 +1,54 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.shardingsphere.shardingproxy.backend.schema;
+
+import com.google.common.collect.Maps;
+import org.apache.shardingsphere.shardingproxy.backend.communication.jdbc.datasource.JDBCBackendDataSource;
+import org.apache.shardingsphere.shardingproxy.config.yaml.YamlDataSourceParameter;
+import org.junit.Test;
+import org.mockito.Mockito;
+import java.lang.reflect.Field;
+import java.util.Map;
+
+import static org.junit.Assert.assertThat;
+import static org.hamcrest.CoreMatchers.is;
+import static junit.framework.TestCase.assertNotNull;
+import static org.mockito.Mockito.when;
+
+public final class LogicSchemaTest {
+
+    @Test
+    public void assertDataSourcesReturned() throws NoSuchFieldException, IllegalAccessException {
```

Comment:
Test case should name as assertXXX, `XXX` is real method name. So for this test case, method name should be assertGetDataSources